docs/semgrep-code/remove-duplicates.md

```diff
@@ -5,26 +5,34 @@ title: Remove duplicate findings
-  - Semgrep Code
-  - Semgrep AppSec Platform
+ - Semgrep Code
+ - Semgrep AppSec Platform
 ---
 
 # Remove duplicate findings
 
 Semgrep scans are performed on both mainline (trunk) and non-mainline branches. The scope of the scan can differ depending on if Semgrep is called on a mainline or non-mainline branch.
 
 <dl>
-    <dt>Full scan</dt>
-    <dd>Scans the repository in its entirety. It is recommended to perform full scans on mainline branches, such as <code>master</code> or <code>main</code>. This scan is performed on a scheduled basis.</dd>
-    <dt>Diff-aware scan</dt>
-    <dd>Diff-aware scans are performed on non-mainline branches, such as in pull requests and merge requests. Diff-aware scans traverse the repository's files based on the commit where the branch diverged from the mainline branch (or diverged from the last commit that was fully scanned)</dd>
+ <dt>Full scan</dt>
+ <dd>Scans the repository in its entirety. It is recommended to perform full scans on mainline branches, such as <code>master</code> or <code>main</code>. This scan is performed on a scheduled basis.</dd>
+ <dt>Diff-aware scan</dt>
+ <dd>Diff-aware scans are performed on non-mainline branches, such as in pull requests and merge requests. Diff-aware scans traverse the repository's files based on the commit where the branch diverged from the mainline branch (or diverged from the last commit that was fully scanned)</dd>
 </dl>
 
-## Remove duplicate findings using Semgrep AppSec Platform
+Regardless of scan type, Semgrep generates a finding whenever a Semgrep rule matches a piece of code.
 
-Regardless of the scope of a scan, Semgrep correlates findings across branches based on their unique fingerprint, automatically deduplicating findings and making it simpler to triage.
+For each finding identified by Semgrep, Semgrep tags the finding with a unique fingerprint that allows Semgrep to track the finding as it exists in a file over time. Semgrep generates this fingerprint by combining and hashing information retrieved during runtime, including file path, rule name, and pattern, and the metavariables substituted into the pattern.
 
-If a finding is fixed in one branch (such as `main`) but open in another (such as `production`), and the code fixes are present in both branches, initiate a scan through your CI job or SCM tool on the branch(es) with open findings to have Semgrep mark the findings as fixed.
+Additionally, Semgrep appends an index value that reflects the number of occurrences of a finding matching the same rule in a single file. The index helps Semgrep determine how two or more findings are related to each other. In other words, a finding with fingerprint `123_0` and a finding with fingerprint `123_1` indicates that the findings were generated from the same rule that matched a given piece of code in the same file.
+
+With this fingerprint, Semgrep can determine if a finding with the same fingerprint already exists. If not, it is considered a new finding.
```

Comment:
Passing thought (feel to shoot this down), but one way users can visualize this is in two places: 
1. The branch status widget on the finding details page
![CleanShot 2024-11-11 at 14 17 07@2x](https://github.com/user-attachments/assets/9f3fa2af-e350-40be-af19-8ed51539eab0)

2. Hovering over the branch associated with the finding on the findings list page.

![CleanShot 2024-11-11 at 14 18 14@2x](https://github.com/user-attachments/assets/a1d1c661-800b-4f7a-b72d-03ed3872187d)

I think it's probably fine to skip (2), but if you'd like something visual for users to associate with, an option would be to point to the branch status widget (1).